source/github-events/on-fragment-load.ts

```diff
@@ -16,7 +16,7 @@ function createFragmentLoadListener(fragmentSelector: string, callback: EventLis
 const diffFileFragmentsSelector = [
 	'include-fragment.diff-progressive-loader', // Incremental file loader on scroll
 	'include-fragment.js-diff-entry-loader', // File diff loader on clicking "Load Diff"
-	'#files_bucket include-fragment', // Diff on compare pages
+	'#files_bucket:not(.pull-request-tab-content) include-fragment', // Diff on compare pages
```

Comment:
#5094 added a new selector for include fragments that is too broad.
It caused `show-whitespace` (and probably other PR files features) to run three or four times when loading the "Files" tab of a PR.

Reply:
D'oh! Didn't realize other features may use that selector as well.

Theoretically `[src*="/compare/"]` can also match a repository named `compare`, though that's rare. We should try finding a selector that only matches on compare page.

Reply:
> We should try finding a selector that only matches on compare page.

The new selector in this PR should do the trick. There's only a handful of features that use `onDiffFileLoad`, and they all target PR files/commits and/or compare pages.